tests/e2e/e2e_test.go

```diff
@@ -116,6 +124,70 @@ var _ = ginkgo.AfterSuite(func() {
 })
 
 var _ = ginkgo.Describe("[Start/Remove/Restart/Add/Stop]", func() {
+	ginkgo.It("can start with blockchain", func() {
+		color.Outf("{{green}}sending 'start' with the valid binary path:{{/}} %q\n", execPath1)
```

Comment:
probably do this like the other tests. see example on `snapshots + blockchain creation` (which should be currently called subnet creation). that is, use ginkgo.By and use common shared variables as in that case.
posible teps:
- can start with blockchain specs
- wait for network to be healthy (you can copy what is on there in snapshots, so as to avoid the LOOP label)
- can create a blockchain with a new subnet id
- wait for network to be healthy
- get subnet id
- can create a blockchain with existing subnet
- wait for network to be healthy
- can save snapshot
- can load snapshot
- wait for network to be healthy
- can create a blockchain with existing subnet

I think this coverts pretty much of the functionality we usually try to get covered when working on this stuff

Reply:
This should all go into a ticket description next time ;-)